.github/workflows/codespace.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and push Docker image
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+env:
+  CR_PAT: ${{ secrets.CR_PAT }}
+  IMAGE_NAME: iog-devx
+  REGISTRY: ghcr.io/input-output-hk
+  DEFAULT_TAG: latest
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: ['x86_64-linux', 'x86_64-darwin', 'aarch64-linux', 'aarch64-darwin']
```

Comment:
Will the platforms other than `x86_64-linux` work?

Reply:
Indeed, no, thanks for having spotted it!